gateway/src/main/java/io/zeebe/gateway/EndpointManager.java

```diff
@@ -419,6 +420,12 @@ public static StatusRuntimeException convertThrowable(final Throwable cause) {
       status = Status.NOT_FOUND.augmentDescription(cause.getMessage());
       Loggers.GATEWAY_LOGGER.debug(
           "Expected to handle gRPC request, but request could not be delivered", cause);
+    } else if (cause instanceof RequestRetriesExhaustedException) {
+      // this error occurs when all partitions have exhausted for requests which have no fixed
+      // partitions - it will then also occur when back pressure kicks in, leading to a large burst
+      // of error logs that is, in fact, expected
+      Loggers.GATEWAY_LOGGER.trace(
```

Comment:
Should we set the status then to resource exhausted? As we do with the BrokerResourceExhausted exception?

Reply:
Previous it was `unexpected error occured` or what was logged and returned?